Doc/tutorial/appendix.rst

```diff
@@ -31,6 +31,32 @@ Typing an interrupt while a command is executing raises the
 :exc:`KeyboardInterrupt` exception, which may be handled by a :keyword:`try`
 statement.
 
+.. _tut-var:
+
+Special ``_`` variable
+-------------------------
+
+There is a special variable ``_`` in interactive mode, which stores the result of the last 
```

Comment:
I guess trailing whitespace means extra space, then there is an extra space at the end of this line